client/manifest.json

```diff
@@ -4,24 +4,33 @@
 	"description": "Self-hosted web IRC client",
 	"start_url": ".",
 	"display": "standalone",
-	"theme_color": "#455164",
-	"background_color": "#455164",
+	"theme_color": "#415363",
+	"background_color": "#415363",
 	"icons":
 	[
 		{
-			"src": "img/touch-icon-192x192.png",
-			"sizes": "192x192",
-			"type": "image/png"
+			"src": "img/logo-grey-bg-120x120px.png",
+			"sizes": "120x120"
 		},
 		{
-			"src": "img/apple-touch-icon-120x120.png",
-			"sizes": "120x120",
-			"type": "image/png"
+			"src": "img/logo-grey-bg-152x152px.png",
+			"sizes": "152x152"
 		},
 		{
-			"src": "img/favicon.png",
-			"sizes": "64x64",
-			"type": "image/png"
+			"src": "img/logo-grey-bg-167x167px.png",
+			"sizes": "167x167"
+		},
+		{
+			"src": "img/logo-grey-bg-180x180px.png",
+			"sizes": "180x180"
+		},
+		{
+			"src": "img/logo-grey-bg-192x192px.png",
+			"sizes": "192x192"
+		},
+		{
+			"src": "img/TODO-logo-grey-bg.svg",
+			"sizes": "193x193"
```

Comment:
Re: SVG, should we keep this?
Also, are we good with the overall map here and in `index.html.tpl` above? (Remember that if an iOS device expects 100x100px, and only 16x16px and 101x101px are provided, the device will go for the 16px version 😠)

Reply:
> if an iOS device expects 100x100px, and only 16x16px and 101x101px are provided, the device will go for the 16px version

Thanks, Apple. You really take design seriously. (Translation: Who could have possibly thought that was the best logic to implement?)

What is the 193px version meant for? I see 192px thrown around in a casual search for common icon dimensions, but not 193.

Reply:
> What is the 193px version meant for? I see 192px thrown around in a casual search for common icon dimensions, but not 193.

Right, it's a placeholder that essentially says "for anything greater than 192px, use the SVG version", which scales better than PNG.

Reply:
That makes perfect sense to me, thanks. I'd say keep the SVG version around, if only _because_ it scales so well in case some device wants a huge logo for some reason. :laughing: